### ISSUE TITLE
UI - VAT as Required Field

### ISSUE THREAD

**Issue by adivated** [energywebfoundation/origin#2188]
This might be a regulatory related item, but the VAT field on an Org is listed as a required field. US Orgs may not have a VAT. I entered 'None' and all, but maybe the field isn't a "required" field?

**Comment by josipbagaric** [comment]
Good catch - `organization.vatNumber` shouldn't really be a required field.
This will require changes on the backend and frontend both.
